models/comments.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection.js");
+
+exports.postComment = (id, payload) => {
```

Comment:
payload isnt the best naming. 
It would be better to destructure the values in the controller, then pass them in as two separate arguments. 
Then you can naming them as body and username in the function. 
Means if you re-use it, its clear that is needed two/three bits of data